### ISSUE TITLE
human-readable __repr__

### ISSUE THREAD

**Issue by mikapfl** [LightForm-group/xrdfit#45]
Hi,

I find it super helpful when working interactively in a notebook if objects have a meaningful `__repr__`, so that I get something helpful if I just let the notebook show me the object. At the moment, xrdfit's objects have the default `__repr__`, which is not very helpful:
```python
>>> peak_params = PeakParams((2.75, 2.95), '(10-10)')
>>> peak_params
<xrdfit.spectrum_fitting.PeakParams at 0x7ff08451b610>
```
For PeakParams, I think you could simply use the `__str__` you already have, maybe with added `<>`:
```python
class PeakParams:
[…]
    def __repr__(self):
        return f'<{str(self)}>'
```

For the other classes, I think not all of them have a useful `__str__` method, `PeakFit` objects for example also have standard `__str__` functions. Especially for `PeakFit` objects I think some `__repr__` could be useful, for example pointing at the PeakParams that were used, or maybe just summarizing the number of peaks that were fit and if the fit was succesful etc.

Cheers,

Mika

**Comment by merrygoat** [comment]
Yes I agree, a good suggestion.

**Comment by merrygoat** [comment]
Fixed in 0e65060062c23dda0970d748e826ac754f752b87
